<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerEdit</class>
 <widget class="QDialog" name="ServerEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Silkroad Server Editor</string>
  </property>
  <widget class="QListWidget" name="lstServers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemove">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Server Editor</string>
   </property>
   <widget class="QLineEdit" name="Hostname">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>gwgt1.joymax.com</string>
    </property>
    <property name="maxLength">
     <number>64</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>iSRO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SilkroadR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>vSRO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rSRO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tSRO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>kSRO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cSRO 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cSRO 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>jSRO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>thSRO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Custom</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hostname/IP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Locale">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>18</string>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Locale:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Port">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>15779</string>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CustomName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="ConnectStartup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect on start up</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>ServerEdit</receiver>
   <slot>LocaleChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>ServerEdit</receiver>
   <slot>Add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemove</sender>
   <signal>clicked()</signal>
   <receiver>ServerEdit</receiver>
   <slot>Remove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstServers</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>ServerEdit</receiver>
   <slot>ServerClicked(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>LocaleChanged(QString)</slot>
  <slot>Add()</slot>
  <slot>Remove()</slot>
  <slot>ServerClicked(QListWidgetItem*)</slot>
 </slots>
</ui>
